docs/user-guide/gateway.md

```diff
@@ -0,0 +1,221 @@
+# Gateway
+
+Not all network services can be provided by the fabric itself due to limitations with modern switch hardware and software.
+The Gateway is designed to provide these more sophisticated network services, such as NAT, PAT, firewalling, and others.
+The tradeoff is that while simple [VPC Peerings](vpcs.md#vpcpeering) offer the full cut-through bandwidth of the fabric, gateway services are limited by the traffic handling capability of the gateway nodes.
+
+
+## Gateway Nodes and Fabric Connectivity
+
+Gateway nodes are connected to the fabric by a set of physical connections that are modeled via Connection objects.
+See the section on [Gateway Connections](connections.md#gateway-connections) for connection requirements and example configuration.
+
+When a Gateway Peering is used to connect two VPCs or externals, the gateway nodes will attract traffic to themselves by advertising the appropriate routes to the fabric.
+In turn, the fabric will use these routes to steer traffic to the gateway so that it can apply the configured peering policy.
```

Comment:
```suggestion
When a Gateway Peering is used to connect two VPCs or externals, the gateway nodes attract traffic to themselves by advertising the appropriate routes to the fabric.
In turn, the fabric uses these routes to steer traffic to the gateway so that the gateway can apply the configured peering policy.
```